index.html

```diff
@@ -1,15 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=<>, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
+       <link rel="stylesheet" href="styles.css">
+       <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
-<body>
-    <span>Hello World!</span>
-    <span>It doesn't work</span>
-    <span>check</span>
-    <span> THIS IS UPDATED TEXT </span>
-</body>
+
+    <body>
+            <div class="basic">
+                <div class="price-card_title">
+                   <span class="title1">Basic</span>
```

Comment:
Try to avoid classes names with numbers:
`title` and `subtitle` looks better for me

Reply:
done in the last commit

Reply:
done in the last commit